subprojects/ide-native/src/main/java/org/gradle/ide/xcode/tasks/GenerateSchemeFileTask.java

```diff
@@ -97,7 +97,9 @@ private void configureLaunchAction(XcodeSchemeFile.LaunchAction action) {
         action.setBuildConfiguration(BUILD_DEBUG);
         for (XcodeTarget xcodeTarget : xcodeProject.getTargets()) {
             XcodeSchemeFile.BuildableReference buildableReference = toBuildableReference(xcodeTarget);
-            action.setBuildableProductRunnable(buildableReference);
+            if (PBXTarget.ProductType.TOOL.equals(xcodeTarget.getProductType())) {
```

Comment:
I see there's also `PBXTarget.ProductType.APPLICATION` and `PBXTarget.ProductType.UNIT_TEST` that sound executable.  Do we not use those?

It might read better if we move this check into `XcodeTarget` and call it `isExecutable` or `isRunnable`.